### PR TITLE
NGN-918 - Renaming the AppID to Processor ID. Including the Processor…

### PR DESCRIPTION
**Pull request by davidsloan**
NGN-918

- Renaming the AppID to Processor ID
- Including the Processor ID on the processor listing

### CODE REVIEW

File: pkg/api/client.go [hunk 1]
```diff
@@ -1420,7 +1420,7 @@ type CreateProcessorPayload struct {
 	ClusterName string `json:"cluster,omitempty" yaml:"cluster"`
 	Namespace   string `json:"namespace,omitempty" yaml:"namespace"`
 	Pipeline    string `json:"pipeline" yaml:"pipeline"`     // defaults to Name if not set.
-	AppID       string `json:"appId,omitempty" yaml:"appId"` //not required
+	ProcessorID string `json:"appId,omitempty" yaml:"appId"` //not required
```
Comment (spirosoik) [inline pkg/api/client.go:1423]: the json tag remains the same though?

**Review by spirosoik — COMMENTED** [review]
one minor comment

File: pkg/api/client.go (hunk 1 above)
Comment (davidsloan) [reply pkg/api/client.go:1423]: This is deliberate, as the API has been designed to retain the old field name.

Comment (davidsloan) [reply pkg/api/client.go:1423]: So we update the field name only for the internal representation, however it goes through to the service via json as before

**Comment by spirosoik** [comment]
@deanlinaras could you have a look to please? overall looks good to me.

**Review by spirosoik — APPROVED** [review]

**Pull request closed without merging**